<launch>

    <!-- Public Params -->
    <param name="vehicleWidth" value="3"/>
    <param name="vehicleLength" value="3.2"/>
    <param name="wheelCircumference" value=".4"/>
    
    <!-- Joy Node -->
    <node pkg="joystick_drivers" type="joy_node" name="joy_node">
    <param name="dev" value="/dev/input/js0">
    </node>

    <!-- Arbitrator -->
    <include file="$(find mdart)/launch/arbitrator.launch">
        <arg name="arbSpeed" value="4.5"/>
    </include>

    <!-- Vehicle Dynamics -->
    <include file="$(find mdart)/launch/vehicledynamics.launch">
        <arg name="vdSpeed" value="10000"/>
        <arg name="vdAccel" value="2"/>
    </include>

    <!-- Serial -->
    <node pkg="mdart" type="serial_gateway" name="serial_gateway"/>

</launch>
